helm_deploy/values-dev.yaml

```diff
@@ -1,30 +1,11 @@
-
-minReplicaCount: 1
-maxReplicaCount: 2
-
-springProfiles: sqs-read
-
-image:
-  repository: quay.io/hmpps/court-list-splitter
-  tag: latest
-  port: 8080
-
-ingress:
-  enabled: true
-  enable_whitelist: true
-  hosts:
-    - host: court-list-splitter-dev.hmpps.service.justice.gov.uk
-      cert_secret: court-probation-dev-cert-secret
-  path: /
-
-env:
-  JAVA_OPTS: "-Xmx512m"
-
-whitelist:
-  office: "217.33.148.210/32"
-  health-kick: "35.177.252.195/32"
-  mojvpn: "81.134.202.29/32"
-  cloudplatform-live1-1: "35.178.209.113/32"
-  cloudplatform-live1-2: "3.8.51.207/32"
-  cloudplatform-live1-3: "35.177.252.54/32"
+generic-service:
+  springProfiles: sqs-read
+
+  ingress:
+    enabled: true
+    enable_whitelist: true
+    hosts:
+      - host: court-list-splitter-dev.hmpps.service.justice.gov.uk
+        cert_secret: court-probation-dev-cert-secret
+    path: /
```

Comment:
i added the values under the generic-service as this is what's needed to work with generic-service